babel/src/main/java/org/apache/calcite/sql/babel/SqlRenameTable.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.calcite.sql.babel;
+
+import org.apache.calcite.jdbc.CalcitePrepare;
+import org.apache.calcite.sql.SqlCall;
+import org.apache.calcite.sql.SqlExecutableStatement;
+import org.apache.calcite.sql.SqlIdentifier;
+import org.apache.calcite.sql.SqlKind;
+import org.apache.calcite.sql.SqlNode;
+import org.apache.calcite.sql.SqlOperator;
+import org.apache.calcite.sql.SqlSpecialOperator;
+import org.apache.calcite.sql.SqlWriter;
+import org.apache.calcite.sql.parser.SqlParserPos;
+import org.apache.calcite.util.ImmutableNullableList;
+
+import java.util.List;
+
+/**
+ * Parse tree for {@code RENAME TABLE} statement.
+ */
+public class SqlRenameTable extends SqlCall implements SqlExecutableStatement {
+  public static final SqlSpecialOperator OPERATOR =
+      new SqlSpecialOperator("RENAME TABLE", SqlKind.OTHER);
+
+  private final SqlIdentifier targetTable;
+  private final SqlIdentifier sourceTable;
+  private final RenameOption renameOption;
+
+  /** Creates a SqlRename. */
+  public SqlRenameTable(SqlParserPos pos, SqlIdentifier targetTable,
+      SqlIdentifier sourceTable, RenameOption renameOption) {
+    super(pos);
+    this.targetTable = targetTable;
+    this.sourceTable = sourceTable;
+    this.renameOption = renameOption;
+  }
+
+  @Override public SqlOperator getOperator() {
+    return OPERATOR;
+  }
+
+  @Override public List<SqlNode> getOperandList() {
+    return ImmutableNullableList.of(targetTable, sourceTable);
```

Comment:
Should we include the renameOption in the operand list?

Reply:
Doesn't it have to be a SqlNode though?

Reply:
Yeah it does. Wasn't sure if it would be worth wrapping it in some sort of SqlNumericLiteral. But that would probably be overkill. We can just leave it as is.